src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -215,6 +215,12 @@
   <data name="FindCommandKey" xml:space="preserve">
     <value>Find</value>
   </data>
+  <data name="FindNextCommandKey" xml:space="preserve">
+    <value>Next search match</value>
+  </data>
+  <data name="FindPrevCommandKey" xml:space="preserve">
+    <value>Previous search match</value>
+  </data>

```

Comment:
VS Code does "Find Next" and "Find Previous".

Personally, I'd prefer "Find next/previous search match" (just prepend 'Find' to what you have now) or "Find next/previous match". But this is up to you. Don't feel strongly about it.